cub/benchmarks/bench/merge_sort/keys.cu

```diff
@@ -127,7 +127,7 @@ void keys(nvbench::state& state, nvbench::type_list<T, OffsetT>)
   thrust::device_vector<nvbench::uint8_t> temp(temp_size);
   auto* temp_storage = thrust::raw_pointer_cast(temp.data());
 
-  state.exec(nvbench::exec_tag::no_batch, [&](nvbench::launch& launch) {
```

Comment:
@gevtushenko the benchmark crashes if I don't put that flag. I guess because the kernel explicitely synchronizes.

Reply:
Do we need that change if we do not use the PDL feature?

Reply:
We would not need that change if PDL is disabled (CTK < 11.8), but it doesn't hurt either (just makes the benchmark a bit faster/more accurate). However, PDL will be enabled unconditionally soon I guess.